pupilos/perfis/MichelDias

```diff
@@ -0,0 +1,16 @@
+Mentor(a) responsável por mim
+
+Ainda não possuo.
+
+Meu Nome
+
+Michel Dias.
+
+Qual meu sonho na carreira?
+
+Iniciar minha carreira na área de desenvolvimento.
+
+Alguns links para me conhecer melhor
+<a href="mailto:michelc.dias@gmail.com">E-mail</a>
```

Comment:
Fala @micheldias, aqui ao invés de usar elementos HTML mesmo, você precisa usar o esquema do Markdown.

Da uma olhada nesse exemplo: 

## Alguns links para me conhecer melhor
[Facebook](https://www.facebook.com/mrgabrielbp)
[Twitter](https://twitter.com/mr_gabrielbp)
[GitHub](https://github.com/gabriel-brito)
[LinkedIn](https://www.linkedin.com/in/gbsantos)

Desse perfil: https://github.com/training-center/mentoria/blob/master/pupilos/perfis/GabrielBrito.md